src/partials/query.editor.html

```diff
@@ -171,29 +171,30 @@
       <info-popover mode="right-absolute" ng-click="showHelp = !showHelp">Click for help</info-popover>
   </div>
   <div class="gf-form--grow" ng-show="showHelp">
+      <!-- TODO: redo after finishing with synthetics -->
       <pre>
-  Format: <code class="query-keyword">timeseries{options}</code>
-      or  <code class="query-keyword">timeseries{options}[aggregation,granularity]</code> <br>
+  Format: <code class="query-keyword">ts{options}</code>
+      or  <code class="query-keyword">ts{options}[aggregation,granularity]</code> <br>
   Options are of the form: property comparator value <br>
   Comparator can be either
     =  (strict equality),
     != (strict inequality),
     =~ (regex equality),
     !~ (regex inequality) <br>
   Templated variables can also be used with [[variable]] or $variable.
-  Example: <code class="query-keyword">timeseries{assetId=[[asset]],metadata.key1 =~ ".*test.*",isStep=1}[average,12h]</code> <br>
+  Example: <code class="query-keyword">ts{assetId=$asset, metadata.key1 =~ ".*test.*",isStep=1}[average,12h]</code> <br>
   Custom functions can also be applied on one or multiple timeseries.
-  Examples: <code class="query-keyword">(timeseries{name=~'.*temp.*'}[avg] - 32) * 5/9</code>
-            <code class="query-keyword">timeseries{} - timeseries{}[avg,24h]</code><br>
+  Examples: <code class="query-keyword">(ts{name=~'.*temp.*'}[avg] - 32) * 5/9</code>
+            <code class="query-keyword">ts{} - ts{}[avg,24h]</code><br>
   Special functions (SUM, MAX, MIN, AVG) can also be applied to all filtered timeseries.
-  Examples: <code class="query-keyword">SUM(timeseries{metadata.type="TEMP"})</code>
+  Examples: <code class="query-keyword">SUM(ts{metadata.type="TEMP"})</code>
               ↪ yields one timeseries that is the sum of all temperature timeseries
-            <code class="query-keyword">MAX(timeseries{}[average]) - MIN(timeseries{}[average])</code>
+            <code class="query-keyword">MAX(ts{}[average]) - MIN(ts{}[average])</code>
```

Comment:
```suggestion
            <code class="query-keyword">MAX(ts{aggregate="average"}) - MIN(ts{aggregate="average"})</code>
```
we won't have square brackets for this anymore, please refer to https://docs.cognite.com/api/playground/#operation/alterTimeSeriesSingle or/and designn doc